DataFormats/ParticleFlowReco/BuildFile.xml

```diff
@@ -17,8 +17,11 @@
 <use name="FWCore/MessageLogger"/>
 <use name="rootcore"/>
 <use name="rootmath"/>
-<flags LCG_DICT_HEADER="classes_1.h classes_2.h"/>
-<flags LCG_DICT_XML="classes_def_1.xml classes_def_2.xml"/>
+<use name="eigen"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/SoATemplate"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<flags ALPAKA_BACKENDS="1"/>
```

Comment:
I suspect that with the latest nightly, this could also be
```suggestion
<flags ALPAKA_BACKENDS="!serial"/>
```
but I have not tried it yet.

So I would go with Matti's suggestion.

Reply:
In the meantime `!serial` is being used by other PRs, so I think
```suggestion
<flags ALPAKA_BACKENDS="!serial"/>
```
is the best way forward.